krcal/map_builder/map_builder_functions.py

```diff
@@ -275,17 +278,16 @@ def check_rate_and_hist(times      : np.array           ,
                                 raising_message = message          )
     return;
 
-def band_selector_and_check(dst       : pd.DataFrame,
-                           boot_map   : ASectorMap,
-                           norm_strat : NormStrategy              = NormStrategy.max,
-                           input_mask : np.array                  = None,
-                           range_Z    : Tuple[np.array, np.array] = (10, 550),
-                           range_E    : Tuple[np.array, np.array] = (10.0e+3,14e+3),
-                           nbins_z    : int                       = 50,
-                           nbins_e    : int                       = 50,
-                           nsigma_sel : float                     = 3.5,
-                           eff_min   : float                      = 0.4,
-                           eff_max   : float                      = 0.6
+def band_selector_and_check(dst        : pd.DataFrame             ,
+                            boot_map   : ASectorMap               ,
+                            range_Z    : Tuple[np.array, np.array],
+                            range_E    : Tuple[np.array, np.array],
```

Comment:
no norm strategy? why?

Reply:
Another comment that didn't get published. I said pretty much the same as @carhc:

Since this is self-contained in ICAROS (it is only used to select the Kr band), it is a free choice for us. We can decide which normalization to use and adapt the energy range in the band selection accordingly. The end user does not see this and neither does it impact the quality of the map.